.github/workflows/release.yml

```diff
@@ -1,63 +1,16 @@
-name: Deployment
+name: Publish Release
 
 on:
-  release:
-    types: [ published ]
+  workflow_dispatch:
+  repository_dispatch:
+    types: [ stdlib-release-pipeline ]
```

Comment:
is it correct to use `stdlib-release-pipeline` dispatch event for the connectors as well?

Reply:
Missed this. Fixed with 91aca74ff7e8b9c62e5381e266cffb5b1f63e68b